src/Aspire.Hosting/Aspire.Hosting.csproj

```diff
@@ -72,6 +72,7 @@
     <InternalsVisibleTo Include="Aspire.Hosting.Tests" />
     <InternalsVisibleTo Include="Aspire.Hosting.Testing.Tests" />
     <InternalsVisibleTo Include="Aspire.Hosting.Containers.Tests" />
+    <InternalsVisibleTo Include="Aspire.Qdrant.Client.Tests"/>
```

Comment:
I need `PasswordGenerator` in 
https://github.com/dotnet/aspire/pull/6057/files#diff-13fe2b0a45a630ecd33dffcd142442c704e675f69070d75928573f60d7a5143a
So I used IVT here.
I did not use `Compile` because `PasswordGenerator` has some dependencies and in this case multiple needs `Compile`. I tried and it got ugly.

Reply:
We can't be using InternalsVisibleTo to these integration tests. The above 3 are "OK" because they are only testing `Aspire.Hosting`. We need to be able to test integrations without having IVT.